models/public/i3d-rgb-tf/i3d-rgb-tf.md

```diff
@@ -0,0 +1,90 @@
+# i3d-rgb-tf
+
+## Use Case and High-Level Description
+
+The `i3d-rgb-tf` is a model for video classification, based on paper ["Quo Vadis, Action Recognition? A New Model and the Kinetics Dataset"](https://arxiv.org/abs/1705.07750). This model use RGB input stream and trained on Kinetics-400\* dataset   . Additionally, this model has initialize values from Inception v1 model pretrained on ImageNet\* dataset.
+
+Originally redistributed as a checkpoint file, was converted to frozen graph.
+
+## Conversion
+
+1. Clone or download original repository:
+    ```
+    git clone https://github.com/deepmind/kinetics-i3d.git
+    ```
+1. (Optional) Checkout the commit that the conversion was tested on:
+    ```
+    git checkout 0667e88
+    ```
+1. Install prerequisites, tested with:
+    ```
+    tensorflow==1.11
+    tensorflow-probability==0.4.0
+    dm-sonnet==1.26
+    ```
+1. Copy [script](./freeze.py) to root directory of original repository and run it:
+    ```
+    python freeze.py
+    ```
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Action recognition |
+| GFLOPs            | 278.981       |
+| MParams           | 12.69         |
+| Source framework  | TensorFlow\*  |
+
+## Accuracy
+
+Accuracy validations performed on validation part of [Kinetics-400\*](https://www.deepmind.com/research/open-source/kinetics) dataset.  Subset consists of 400 randomly chosen videos from this dataset.
+
+| Metric | Converted Model | Converted Model (subset 400) |
+| ------ | --------------- | ---------------------------- |
+| Top 1  | 65.96%          | 67.0%                        |
+| Top 5  | 86.01%          | 88.7%                        |
+
+## Input
+
+### Original Model
+
+Video clip, name - `Placeholder`, shape - `1,79,224,224,3`, format is `B,F,H,W,C`, whereL
+- `B` - batch size
+- `F` - frames
+- `H` - height
+- `W` - width
+- `C` - channel
+
+Channel order is `RGB`. Mean value - 127.5, scale value - 127.5.
+
+### Converted Model
+
+Video clip, name - `Placeholder`, shape - `1,79,3,224,224`, format is `B,F,C,H,W`, whereL
+- `B` - batch size
+- `F` - frames
+- `C` - channel
+- `H` - height
+- `W` - width
+
+
+Channel order is `BGR`.
```

Comment:
it is not true, MO currently do not support channels reversiong for 5D tensors, so 
```suggestion
Channel order is `RGB`.
```

Reply:
13:01:28  [ WARNING ]  Network has 1 inputs overall, but only 0 of them are suitable for input channels reversing.